lib/src/main/java/graphql/nadel/engine/transform/hydration/NadelHydrationTransform.kt

```diff
@@ -187,88 +273,99 @@ internal class NadelHydrationTransform(
 
         // Do nothing if there is no hydration instruction associated with this result
         if (instructions.isEmpty()) {
-            return emptyList()
+            return NadelPreparedHydration {
```

Comment:
This code should mainly be the same. All the code as just be wrapped in a `NadelPreparedHydration`